### PR TITLE
Add yaml file describing Alluxio component owners

### PR DESCRIPTION
**Pull request by tieujason330**

### CODE REVIEW

File: dev/github/component_owners.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+# available at www.apache.org/licenses/LICENSE-2.0
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied, as more fully set forth in the License.
+#
+# See the NOTICE file distributed with this work for information regarding copyright ownership.
+#
+# Mapping of Alluxio component to Github user(s) that will be automatically assigned for pull requests
+# Map<string, string[]>
+component-owner-mapping:
```
Comment (Xenorith) [inline dev/github/component_owners.yaml:13]: add a default component as a fallback, which helps handle the case where someone adds a new directory but did not update this mapping?

File: dev/github/component_owners.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  assembly: 
+    - yuzhu
+  bin: 
+    - LuQQiu
+  build: 
+    - yuzhu
+  conf: 
+    - bf8086
+  core: 
+    - gpang
```
Comment (Xenorith) [inline dev/github/component_owners.yaml:23]: change to ggezer

File: dev/github/component_owners.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  docs: 
+    - apc999
+  examples: 
+    - LuQQiu
+  integration: 
+    - jiacheliu3
+  job: 
+    - bradyoo
+  libexec: 
+    - gpang
```
Comment (Xenorith) [inline dev/github/component_owners.yaml:37]: change to ggezer

**Review by Xenorith — CHANGES_REQUESTED** [review]

File: dev/github/component_owners.yaml (hunk 1 above)
Comment (tieujason330) [reply dev/github/component_owners.yaml:14]: updated

**Review by Xenorith — APPROVED** [review]

**Comment by tieujason330** [comment]
alluxio-bot, merge this please

**Pull request merged**